vars/Debian.yml

```diff
@@ -10,5 +10,5 @@ __apache_packages:
   - apache2-utils
 
 apache_ports_configuration_items:
-  - regexp: "^Listen "
+  - regexp: "^Listen {{ apache_listen_port }}$"
```

Comment:
It would.  My goal was to avoid clobbering listening ports set by a different playbook.  

Would adding a flag to distinguish between the two cases work for you?  Something like `apache_single_instance = True` by default with old functionality, and setting `apache_single_instance = False` to limit altering lines that aren't set by the current playbook.